src/CTA.Rules.PortCore/SolutionPort.cs

```diff
@@ -132,6 +132,15 @@ private void InitRules(List<PortCoreConfiguration> solutionConfiguration, List<A
             var projectTypeFeatureDetector = new FeatureDetector();
             var projectTypeFeatureResults = projectTypeFeatureDetector.DetectFeaturesInProjects(analyzerResults);
 
+            if (projectTypeFeatureResults.Values.Any(f => f.PresentFeatures.Any()))
+            {
+                var reportGenerator = new FeatureDetectionResultReportGenerator(_context, projectTypeFeatureResults);
+                reportGenerator.GenerateFeatureDetectionReport();
+
+                LogHelper.LogInformation("Generating FeatureDetection Report");
+                LogHelper.LogError($"{Constants.MetricsTag}: {reportGenerator.FeatureDetectionResultJsonReport}");
```

Comment:
A few questions about this:
1) Should this be part of the report we generate for the customer?
2) This means this tag will be added twice to the log.. Any issues with that being processed?

I guess the real question here is why is this separate, instead of the same report?

Reply:
Per our discussion, I will add these metrics to the analysis report